cmd/publishing-bot/config/rules.go

```diff
@@ -31,7 +31,7 @@ func (c Dependency) String() string {
 
 // Source of a piece of code
 type Source struct {
-	Repository string `yaml:"repository"`
+	Repository string `yaml:"repository,omitempty"`
```

Comment:
Why is this omitempty?

Reply:
Running the script generates a diff with these fields, so would be good to add omitempty for them:
```
skip-source-branches: []
skip-godeps: false
skip-gomod: false
skip-tags: false
```